src/main/java/com/glodblock/github/client/me/FluidRepo.java

```diff
@@ -0,0 +1,260 @@
+ *
+ * Applied Energistics 2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Applied Energistics 2.  If not, see <http://www.gnu.org/licenses/lgpl>.
+ */
+
+package com.glodblock.github.client.me;
+
+import appeng.api.AEApi;
+import appeng.api.config.*;
+import appeng.api.storage.data.IAEFluidStack;
+import appeng.api.storage.data.IAEItemStack;
+import appeng.api.storage.data.IItemList;
+import appeng.client.gui.widgets.IScrollSource;
+import appeng.client.gui.widgets.ISortSource;
+import appeng.client.me.ItemRepo;
+import appeng.core.AEConfig;
+import appeng.items.storage.ItemViewCell;
+import appeng.util.ItemSorters;
+import appeng.util.Platform;
+import appeng.util.item.AEItemStack;
+import appeng.util.prioitylist.IPartitionList;
+import com.glodblock.github.common.item.ItemFluidDrop;
+import cpw.mods.fml.relauncher.ReflectionHelper;
+import net.minecraft.item.ItemStack;
+
+import javax.annotation.Nonnull;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.regex.Pattern;
+
+
+public class FluidRepo extends ItemRepo {
+
+    private final IItemList<IAEItemStack> list = AEApi.instance().storage().createItemList();
+    private final ArrayList<IAEItemStack> view = new ArrayList<IAEItemStack>();
+    private final ArrayList<ItemStack> dsp = new ArrayList<ItemStack>();
+    private final IScrollSource src;
+    private final ISortSource sortSrc;
+
+    private int rowSize = 9;
+
+    private String searchString = "";
+    private IPartitionList<IAEItemStack> myPartitionList;
+    private String innerSearch = "";
+    private String NEIWord = null;
+    private boolean hasPower;
+
+    public FluidRepo(final IScrollSource src, final ISortSource sortSrc) {
+        super(src, sortSrc);
+        this.src = src;
+        this.sortSrc = sortSrc;
+    }
+
+    public IAEItemStack getReferenceItem(int idx) {
+        idx += this.src.getCurrentScroll() * this.rowSize;
+
+        if (idx >= this.view.size()) {
+            return null;
+        }
+        return this.view.get(idx);
+    }
+
+    public ItemStack getItem(int idx) {
+        idx += this.src.getCurrentScroll() * this.rowSize;
+
+        if (idx >= this.dsp.size()) {
+            return null;
+        }
+        return this.dsp.get(idx);
+    }
+
+    void setSearch(final String search) {
+        this.searchString = search == null ? "" : search;
+    }
+
+    public void postUpdate(final IAEItemStack is) {
+        IAEFluidStack fluid = ItemFluidDrop.getAeFluidStack(is);
+        IAEItemStack i = AEItemStack.create(is.getItemStack().setStackDisplayName(fluid.getFluid().getLocalizedName()));
```

Comment:
you make a display only drop item like the fluid packet, so the name will look like normal and no strange tooltip